packages/changed-elements-react/src/utils/utils.ts

```diff
@@ -0,0 +1,57 @@
+/*---------------------------------------------------------------------------------------------
+* Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+* See LICENSE.md in the project root for license terms and full copyright notice.
+*--------------------------------------------------------------------------------------------*/
+
+export async function* splitBeforeEach<T, U>(
+  iterable: AsyncIterable<T>,
+  selector: (value: T) => U,
+  markers: U[],
+): AsyncGenerator<T[]> {
+  let accumulator: T[] = [];
+  let currentMarkerIndex = 0;
+  for await (const value of iterable) {
+    if (currentMarkerIndex !== markers.length && selector(value) === markers[currentMarkerIndex]) {
+      yield accumulator;
+      accumulator = [];
+      ++currentMarkerIndex;
+    }
+
+    accumulator.push(value);
+  }
+
+  yield accumulator;
+}
+
+
+export async function* flatten<T>(iterable: AsyncIterable<T[]>): AsyncGenerator<T> {
```

Comment:
Seems like this is currently unused? The only usage I can see is in `VersionCompareSelectWidget.tsx` and it's currently using the flatten function defined in that same file. So either remove this or remove the one in the VersionCompareSelectWidget.

Reply:
Same with the other ones here, maybe you forgot to replace them?

Reply:
They are now being imported into the widget.